lang/java/java.talon

```diff
@@ -30,24 +41,16 @@ type {user.code_type} array:
     insert(user.code_type)
     user.code_operator_subscript()
 
-[state] {user.java_access_modifiers}: 
+[state] {user.java_access_modifiers}:
     insert(user.java_access_modifiers)
     key("space")
 
-[state] {user.java_modifiers}: 
+[state] {user.java_modifiers}:
     insert(user.java_modifiers)
-    key("space") 
+    key("space")
 
 op array:
     user.code_operator_subscript()
 
 op new:
     insert("new ")
-
-# Methods    
```

Comment:
Noting that this addresses #662 (replaces java specific function commands with the generic ones).